programs/stake/src/stake_state.rs

```diff
@@ -704,6 +704,9 @@ pub fn split(
         StakeStateV2::Stake(meta, mut stake, stake_flags) => {
             meta.authorized.check(signers, StakeAuthorize::Staker)?;
             let minimum_delegation = crate::get_minimum_delegation(&invoke_context.feature_set);
+            let clock = invoke_context.get_sysvar_cache().get_clock()?;
+            let status = get_stake_status(invoke_context, &stake, &clock)?;
+            let is_active = status.effective > 0;
```

Comment:
we need to feature gate these to prevent inconsistent returns during upgrade, right?

Reply:
I don't think so. Is there some reason I'm missing that the sysvars wouldn't be available?

Reply:
shouldn't be. but the methods are fallible so... :shrug:

Reply:
On the one hand, if we have indeterminism from sysvar imports, we have much bigger problems independent of this PR. On the other, the only other sysvars pulled into the stake program are in redelegate, so feature-gated and not switched on yet. I’ll wrap these when I’m back this evening

Reply:
Done